wallet-services/src/main/java/org/exoplatform/wallet/service/WalletAccountServiceImpl.java

```diff
@@ -517,25 +517,11 @@ public void setInitializationStatus(String address,
       throw new IllegalStateException(CAN_T_FIND_WALLET_ASSOCIATED_TO_ADDRESS + address);
     }
 
-    if (!isUserRewardingAdmin(currentUser)) {
```

Comment:
This code should not be removed but refactored to consider the case when a user wants to delete his own wallet and recreate a new one